### ISSUE TITLE
Promotion and Marketing ideas

### ISSUE THREAD

**Issue by aubergine10** [DoESLiverpool/somebody-should#480]
At the organisers' meeting, the subject of marketing, website and general vision of DoES was raised multiple times, so that's something that's now actively being looked in to.

> If you have any input to add, please do so we can better guide the next steps. Please also mention how you use DoES so we can get better idea of how different "roles" (for want of better word) perceive things.

As a newbie maker, who might also be interested in hot-desking, here's my thoughts:

* The hot-desking and maker aspects of DoES are blurred - it's confusing for newbies trying to work out what DoES does
    * From a marketing perspective, I think the two should be considered as separate entities, so they can be more clearly defined and also illustrate how they benefit each other.
    * The website homepage should probably reflect this
* LoRaWAN - we have two gateways... invaluable to anyone wanting to develop LoRa based products / services
    * DoES experience with microcontrollers and telemetry in general should be more visible
* Machines - 3D printers, CNC, laser cutter, etc. - we need clearer information on what they are, what they do and examples of what people have done with them
    * These are a major USP in themselves IMO, as is the community of makers who can assist with using them
* Completed projects - it takes too long to wade through the weeknotes to find what's been made at DoES
    * maybe we should have a page that highlights some of the successful projects?
* There are loads of groups that meet at DoES that most of us don't know about
    * At the very least, they should be listed on calendar (and also linked to something that provides more detail)
    * Maybe a page for each on the website for each group?
* There's obviously a wide range of skills within the community - but who has what skill?
    * Maybe a few key people could be listed on website as initial points of contact for specific areas of expertise?

**Comment by ajlennon** [comment]
You had me at "cake garden"

**Comment by ajlennon** [comment]
btw. The Google calendar link is here. I have it plugged into Thunderbird and it seems to work well

https://calendar.google.com/calendar/embed?src=doesliverpool.com_4f906tbj39irupt6crbjhi998k%40group.calendar.google.com&ctz=Europe/London

**Comment by aubergine10** [comment]
Many of the events on that calendar don't provide any additional detail about the event - for example, what is abstraction workshop? What is ScouseFP? Also, I was told recently that there are about 15 groups meeting at DoES - are some missing from calendar?

**Comment by ajlennon** [comment]
The abstraction workshop is to do with making better product designs, and ScouseFP is apparently a group of deviants who do things with LISP

**Comment by martimorta** [comment]
About communication, where is best to ask a question or share something? mail list, discourse, slack?

**Comment by aubergine10** [comment]
Depends what you're sharing. I usually use the [google group](https://groups.google.com/forum/#!forum/does-liverpool) or twitter (create a tweet containing `@DoESLiverpool #weeknotes` and it will end up in next weeks' week notes which are posted to the website and google group and probably other places too).

**Comment by jackie1050** [comment]
I actually went to that ScouseFP meetup after clicking on the link in Weeknotes, despite no previous interest in functional programming (really good talk on universal register machines).  Obviously some info is getting through...

It would be great to have more information on the calendar, and I believe links are generally added where event organisers provide them. I think part of the problem is the dual role of the calendar in booking spaces and publicising events. To some extent, it should be up to event organisers to decide how much info/what links they want to provide to the general public.

**Comment by aubergine10** [comment]
I've recently done some updates to the wearables group wiki page - what do you think of this format for group overview: https://github.com/DoESLiverpool/wiki/wiki/Liverpool-Wearable-Tech-Group

**Comment by jackie1050** [comment]
That’s great Guy – very clear and it’s good to include more photos.

I know Laura has plans to add extra content too.

I’ve taken out a reference to the sewing club on the wearables group page. Will need someone with access to events calendar to remove upcoming sewing club meetings, and there’s still a wiki page which probably should be removed or updated to say that they’ve moved on.

**Comment by dazz-ling** [comment]
A social media gallery of some of of the thing made in the workshop would make it easier to help spread the word.

**Comment by aubergine10** [comment]
I'll find out where the sewing club went and update their page to point people in right direction.

I'll remove sewing club ref from wearables page.

I don't have access to calendar so can't update that, but will find out who can.

As for social media gallery, Instagram and twitter are already used by DoES users, but it would be good to get a list of all the key social media outlets and work out some sort of strategy (even if it's just what hashtag to use) to help with discoverability.

**Comment by johnmckerrell** [comment]
I’ve removed the sewing club from the calendar, I’m not averse to adding the new date and venue once we find it out.

**Comment by stevesparrow** [comment]
@DoESsean  is heading up some stuff looking at promotion of Does, and looking at the website stuff. Just flagging him into this, and mentioning it. This will all be really useful for him:)

**Comment by amcewen** [comment]
@aubergine10 The calendar does have a link to the event (or the general meetup page/website for the group) in the description, where we know it - Google Calendar just doesn't do a very good job of showing that :-/  That's where the [weeknotes script](https://github.com/DoESLiverpool/weeknote-generator) expects it to be.  I [blogged about my issues with calendars](http://www.mcqn.net/mcfilter/archives/computers/whats_wrong_with_calendars.html) recently, should anyone want to know more about calendars than is sensible... ;-)  It's something I'd like to improve, but I don't _think_ it's a quick fix (BICBW).

Curating a set of "cool things made in the space" for the website would be cool.  The weeknotes should give a good starting point for finding suitable things (and will generally give you the details of who to talk to if you want to do something a bit more wordy around them - or someone to ask to write something more wordy about it).  Could be a semi-regular feature with a blog post written for each item, and a page to show everything in that category?  That'd give us something to promote each time there was a new item added, and a place that newcomers could find a subset of Epic Sh** made by the community.

**Comment by SayBeano** [comment]
I'd really like to see a cabinet of "Epics" in the hall or in the main space. 

People, not including our community, don't always get a chance to go in the workshop or in Dinky. Granted the workshop shows more "aborts" than Epics.

I do like the idea of an online gallery, but these never properly display the scale nor are they tangible.

An further idea for a cabinet would be a sign saying "Ask for the key." and as Epics are theoretically reproducible, the could be sold if anyone sees something they like.

**Comment by DefProc** [comment]
On the events front, I've just changed the [calendar page](http://doesliverpool.com/calendar) to show the agenda view for each calendar by default. IMO that's a bit easier to see the actual events, even if it does obscure the regular repetition.

**Comment by aubergine10** [comment]
@amcewen I have a license for a nifty calendar / events plugin for WP that I'm not using. Maybe something that can be investigated more when we take a look at website?

As for a bestiary of cool stuff, I think we should aim towards having a blog post for each cool thing and then we can display that category of posts Pinterest-style on a specific page. I have WP themes and plugins that can help with that sort of thing, but obviously the big task will be creating and correctly-dating the blog posts etc.

**Comment by SayBeano** [comment]
Avoid blogposts for this bestiary, IYAM, and use portfolios instead.

https://en.support.wordpress.com/portfolios/

**Comment by aubergine10** [comment]
Any specific reason to use portfolios over tagged/categorised blog posts?

**Comment by SayBeano** [comment]
They're more maliable from a data sense and easier to showcase within wordpress.

Create a page and add specific items to it.
Less fidgiting with dates too.
Multile ways to sort.
Multiple ways to display.
Create a page with multiple catergories and more than one shortcode.
Create multiple pages displaying specific data.
Blog catergories/tags presume a knowledge of existing tags.
Users interacting with tags on a blog isn't very pinterest.
Portfolios follow links/linked data so you can do a page of tags if you wanna.

They're just better for this sort of thing, IYAM.

**Comment by aubergine10** [comment]
Some related tickets: #383 and #306

**Comment by ajlennon** [comment]
I've been dipping a tentative toe into the Patreon model in recent months.

I love the theory of it, direct support to those people and projects you wish to see succeed, but I'm not yet entirely convinced this can succeed in the middle to long term. It seems quite fragile.

That aside, I've been starting to support stuff that interests me and most recently have decided to give the maintainer of OctoPrint / OctoPi $1 + fees a month. I think I can manage $12 + fees a year and today I really like what she's doing and want her to do more.

I was a little surprised to see on her Patreon page that she's pulling in $6K a month from this project. 

ref: https://www.patreon.com/foosel/overview

I've heard some comments around the DoES community about Patreon - am I thinking of you @DoESsean - and this pushes me to think there's value in the channel.

When you become a patron of a creator or proejct you get access to stuff, sometimes early access.

If the above is interesting are we in a position to provide early access to "something" or a feed or whatever to DoES supporters? Who would have the time and energy to build and maintain a supporter-community in this way? 

It seems to me if a great project like OctoPrint can pull in $6K globally from supporters then there's something here that is worth looking into for DoES?

**Comment by DoESsean** [comment]
Yeah, I briefly raised Patreon before, and do think it's something that is
worthwhile our looking into.

As I remember, there were some ethical issues around the people that run
it, or something? I may be wrong, there's mostly some ethical issues around
everything I bring up :D

**Comment by ajlennon** [comment]
Yeah I remember and I remember commenting on the RealSexyCyborg ethical stuff.

I decided for now for me it's better to support people I like on a platform I'm not sure about than not support people I like.

**Comment by Robotorium** [comment]
I think probably we should finish the crowdfunding rewards before we try
something like this? What needs to happen next for the bricks etc?

Op di 26 feb. 2019 om 13:41 schreef Alex Lennon <notifications@github.com>:

**Comment by ajlennon** [comment]
Oooh. Tom was thinking of running a casting workshop in the space at the back of the Tapestry Building.

If that was to work out I’m wondering if we could cast some brick fascias (no iPhone not cast some fascists)

**Comment by amcewen** [comment]
@ajlennon your enthusiasm for learning new things is wonderful.  However, it would be great if you could work on your spotting-if-this-will-help-us-finish-things-or-if-it-will-distract-and-delay-things, because it looks to me like in this instance your suggestion falls into the second of those options...

**Comment by ajlennon** [comment]
Not at all. The brick issue has been sitting there waiting to be dealt with for too long.

I’m attempting to put a date on an event where we deal with it
